sources/us/nc/craven.json

```diff
@@ -16,7 +16,8 @@
             "PASTDR",
             "PASTNA",
             "PASTAB"
-        ]
+        ],
+        "postcode": "zip"
```

Comment:
Same thing, `zip` refers to the owners' mailing zip code.  However, `PACITY` contains the situs city if you'd like to add that.